integration/airflow/README.md

```diff
@@ -206,6 +209,20 @@ t1 = DataProcPySparkOperator(
             f"-javaagent:{jar}={os.environ.get('OPENLINEAGE_URL')}/api/v1/namespaces/{os.getenv('OPENLINEAGE_NAMESPACE', 'default')}/jobs/{job_name}/runs/{{{{lineage_run_id(run_id, task)}}}}?api_key={os.environ.get('OPENLINEAGE_API_KEY')}"
         dag=dag)
 ```
+
+Airflow 2.0+:
+
+```python
+t1 = DataProcPySparkOperator(
+    task_id=job_name,
+    #required pyspark configuration,
+    job_name=job_name,
+    dataproc_pyspark_properties={
+        'spark.driver.extraJavaOptions':
+            f"-javaagent:{jar}={os.environ.get('OPENLINEAGE_URL')}/api/v1/namespaces/{os.getenv('OPENLINEAGE_NAMESPACE', 'default')}/jobs/{job_name}/runs/{{{{macros.OpenLineagePlugin.lineage_run_id(run_id, task)}}}}?api_key={os.environ.get('OPENLINEAGE_API_KEY')}"
```

Comment:
yuck. Is that the only way to reference the macro? Can't be shortened at all?

Reply:
`macros.OpenLineagePlugin.lineage_run_id(run_id, task)` - this is the macro, and can be shortened by changing macro name from `OpenLineagePlugin` and function name `lineage_run_id` to something shorter 🙂

Reply:
Yeah, it was specifically the `macros.OpenLineagePlugin.lineage_run_id` bit that I found yucky. It's Airflow's convention, and I guess we can't really change that.

We _might_ consider renaming the `OpenLineagePlugin` class to just `OpenLineage` - it's already in a package named `openlinage.airflow.plugin` - there's a lot of redundancy there 😂. But save for another PR